packages/extension/package.json

```diff
@@ -39,6 +37,7 @@
     "run-sequence": "1.2.2",
     "sinon": "7.3.2",
     "sinon-chai": "3.3.0",
+    "snap-shot-it": "7.9.3",
```

Comment:
why was this added?

Reply:
oops. i was originally going to use it to fix some tests in the extension, but ended up not and forgot to remove this. fixed